templates/init.erb

```diff
@@ -107,13 +113,13 @@ stop() {
 }
 
 status() {
-  if [ ! -f "$PID_FILE" ]; then
+  if ! [ -f "$PID_FILE" ]; then
```

Comment:
to me, `if ! [` denotes `unless [`, whereas `if [ !` denotes: ATTENTION: YOU HAVE TO THINK ABOUT LOGIC NOW